lib/pdk/version.rb

```diff
@@ -1,4 +1,4 @@
 module PDK
-  VERSION = '1.19.0.pre'.freeze
+  VERSION = '1.19.1.pre'.freeze
```

Comment:
```suggestion
  VERSION = '1.19.0.pre'.freeze
```

Reply:
I think the rake build task updates that automatically.  Anyways I set it back to 0.